pkg/client/connector/install.go

```diff
@@ -455,30 +455,30 @@ var agentNotFound = errors.New("no such agent")
 // is installed alongside the proper deployment.  In doing that, it also ensures that
```

Comment:
This comment needs an update. It's no longer "deployment" and now the function also returns the object's kind.